### ISSUE TITLE
File Path Completion Inserts The Whole Relative After ./ or ../

### ISSUE THREAD

**Issue by ahmadfebrianto** [microsoft/pylance-release#1490]
When using file path completion/suggestion to import/read a file, it inserts the whole relative path into the path string while it should only insert the remaining path string after we prepend the path with ``./`` or ``../`` notation.

## Environment data

-   VS Code version: 1.57.1
-   Jupyter Extension version: v2021.6.999406279
-   Python Extension version: v2021.6.944021595
-   OS (Linux Mint) and version: 20 (Ulyana)
-   Python: 3.8.5 (64-bit)
-   Type of virtual environment used: (N/A)
-   Jupyter server running: Local

## Steps to reproduce:

1. Create one ``ipynb`` file and one ``csv`` file in the same directory.
2. Open the ipynb file in VS Code.
3. Try reading/importing the second file using ``open()`` or ``pd.read_csv()``.
4. Write ``./`` in the parentheses and click ``CTRL + Space`` until the suggestion list appears.
5. Scroll down to the ``csv`` file from the suggestion list. 
6. Press ``ENTER`` or ``TAB ->``

https://user-images.githubusercontent.com/52089094/123132442-f36b6d80-d478-11eb-9fc4-8e9ff2f6f62e.mp4

**Comment by ahmadfebrianto** [comment]
. So should I post this issue to Pylance then?

> I believe this would be pylance's fault? The TextEdits coming back for the autocomplete aren't skipping the already existing text?

Yeah, it looks like that. I was wondering if this was intentional and they had assigned some shortcut to get the correct path.

**Comment by jakebailey** [comment]
No, I don't think this is us. We do not do file path completions at all. I thought those were entirely provided by Jupyter.
